covidapi/import_data.py

```diff
@@ -0,0 +1,33 @@
+import csv
+from db.models import DailyReport
+from db.database import SessionLocal, engine, Base
+from datetime import datetime
+
+
+def main():
+    print("Importing data into db")
+
+    Base.metadata.create_all(engine)
+    db_instance = SessionLocal()
+
+    with open('02-29-2020.csv') as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=',')
+        next(csv_reader, None)  # skip the headers
+
+        for row in csv_reader:
+            dr = DailyReport(
+                province_state=row[0],
+                country_region=row[1],
+                last_update=datetime.fromisoformat(row[2]),
+                confirmed=row[3],
+                deaths=row[4],
```

Comment:
I was wondering if we need to explicitly convert to int here, but sqlalchemy seems to take care of it.